src/add-link.js

```diff
@@ -0,0 +1,8 @@
+document.getElementById('add-link').addEventListener('click', () => {
```

Comment:
Since this is to support the demo (rather than the package itself) can we move it out of `src` to `demo/add-link.js`? If that doesn't easily work with our existing setup feel free to inline it too.